src/lib/Api/api.ts

```diff
@@ -145,9 +146,22 @@ export default class Api {
       });
     }
 
+    if (opts.tokenAddresses) {
+      opts.tokenAddresses.forEach(item => {
+        if (!utils.isAddress(item)) {
+          log.throwMissingArgumentError(Logger.message.invalid_token_address, {
```

Comment:
`MISSING_ARGUMENT` doesn't seem like the right error code.  The argument was provided, it's just invalid.  `INVALID_ARGUMENT` seems more appropriate:

```
log.throwArgumentError(Logger.message.invalid_token_address, 'tokenAddresses', item, {
  location: Logger.location.SDK_GETNFTS,
});
```

But since `throwMissingArgumentError` is used similarly in this file, it can be kept for consistency if you prefer.

Reply:
I agree, i ll change it for this one but we should refactor at some point